src/components/StatDashboard.vue

```diff
@@ -15,6 +16,7 @@ import StatPanel from '@/components/StatPanel.vue';
 import DateWidget from '@/components/widgets/DateWidget.vue';
 import MoonWidget from '@/components/widgets/MoonWidget.vue';
 import GraphqlWidget from '@/components/widgets/GraphqlWidget.vue';
+import TemperatureWidget from './widgets/TemperatureWidget.vue';
```

Comment:
@ is een shortcut voor de src directory. Graag hier gebruik van maken.

Reply:
Done